config/cloud.cfg.tmpl

```diff
@@ -107,10 +119,15 @@ cloud_config_modules:
 {% endif %}
 {% if variant not in ["photon"] %}
  - ssh-import-id
- - keyboard
  - locale
+{% if variant not in ["rhel"] %}
+ - keyboard
```

Comment:
This inadvertently changes the order that modules are run, putting keyboard after locale. Should we move this back above locale?

Reply:
Moving it back, thank you